mrmustard/lab_dev/circuit_components.py

```diff
@@ -151,11 +150,32 @@ def bargmann(self) -> tuple:
         r"""
         The Bargmann parametrization of this circuit component, if available.
         """
-        if not isinstance(self.representation, Bargmann):
-            raise ValueError(
+        try:
+            return self.representation.triple
+        except AttributeError as e:
+            raise AttributeError(
                 f"Cannot compute triple from representation of type ``{self.representation.__class__.__qualname__}``."
-            )
-        return self.representation.triple
+            ) from e
+
+    def quadrature(self) -> tuple | ComplexTensor:
```

Comment:
Is this tested somewhere?

Reply:
The map is tested here, but for sure we could beef up the tests of this and other things.
https://github.com/XanaduAI/MrMustard/blob/3f804033e0ed9c505677768534b615daf83c15b5/tests/test_lab_dev/test_circuit_components_utils.py#L165C7-L165C18

Reply:
Yes could be good to have a test of this.